front/src/modules/rollingStock/components/RollingStockSelector/RollingStockCurve.tsx

```diff
@@ -106,17 +119,37 @@ function Legend(props: {
           tabIndex={0}
           onClick={() => changeCurveState(curve.id)}
         >
-          {curve.mode}
-          {curve.comfort !== 'STANDARD' ? comfort2pictogram(curve.comfort) : null}
+          {isOnEditionMode && (
+            <>{showPowerRestriction ? curve.power_restriction : curve.electrical_profile_level}</>
```

Comment:
follow warnings and use <div> <text> and not a fragment

Reply:
Done :)